### PR TITLE
Bakoshi

### PR DESCRIPTION
**Pull request by Kadunapikin**
Revisited rock, paper, scissors game and added more functionalities and styles to the UI. Find attached picture of the user interface bellow.

Rock paper scissors interface

### CODE REVIEW

File: RockPaperScissors.js [hunk 1]
```diff
@@ -1,59 +1,99 @@
...
+    const randomNumber = choices[Math.floor(Math.random() * choices.length)];
+    computerChoice = randomNumber
+    computerChoiceDisplay.innerHTML = computerChoice
 }
 
-game();
\ No newline at end of file
+ const getResult = () => {
+    switch (userChoice + computerChoice) {
+    case 'rockscissors':
```
Comment (zizoh) [inline RockPaperScissors.js:70]: Can have a list containing win combinations: `rockscissors`, `paperrock` and `scissorspaper`, lose combinations and draw combinations. Then you can check if userChoice + computerChoice is contained in those lists.

File: RockPaperScissors.js [hunk 2]
```diff
@@ -1,59 +1,99 @@
...
+    if (clickCount >= 5) {
+        disableButtons();
+    }
 }
+possibleChoices.forEach(possibleChoice => possibleChoice.addEventListener('click', handleClick));
 
-function computerChoice() {
-    return choices[Math.floor(Math.random() * choices.length)];
-}
+function winner(ps, cs, rn) {
```
Comment (zizoh) [inline RockPaperScissors.js:46]: Can be renamed checkWinner or showWinner. Or something similar.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+.choices-display, .score-board {
+    display: flex;
+    background-color: bisque;
+    color: rgb(2, 2, 31);
+    justify-content: space-around;
+    padding: 30px;
+}
+
+.score-board {
+background-color: beige;
```
Comment (zizoh) [inline style.css:42]: This needs formatting.

File: RockPaperScissors.html [hunk 4]
```diff
@@ -5,8 +5,37 @@
...
+    <div class="choices">
+        <button class="choice" id="rock">Rock</button>
+        <button class="choice" id="paper">Paper</button>
+        <button class="choice" id="scissors">Scissors</button>
+    </div>
+    <div class="decide">
+        <h3>Please make your choice to play the game</h3>
+    </div>
     
 <script src="RockPaperScissors.js"></script>
```
Comment (Phildeoner) [inline RockPaperScissors.html:40]: Adjust this indentation

**Review by Phildeoner — COMMENTED** [review]
Hi @Kadunapikin 👋👋, Wow you did a great Job, well-done👍👍, I love the interface very simplified.
I have a suggestion, what do you think about creating a function to automatically refresh the page instead of the player manually refreshing after exhausting the desired rounds? 
Nice work once again and Happy Coding😊😊

File: RockPaperScissors.html (hunk 4 above)
Comment (Kadunapikin) [reply RockPaperScissors.html:40]: Done

**Review by Phildeoner — APPROVED** [review]
Hi @Kadunapikin, you did a great job nothing more to say, your work is complete you can proceed to merge👌👍.
Happy coding☺️☺️

**Review by zizoh — APPROVED** [review]

**Pull request merged**